docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: docs
+title: Importing projects
+description: Documents
+keywords: importing, directory, archive, configuring, cloud, microservices, application, Eclipse, MicroProfile, Java, Spring, Node.js, Swift, Maven, Dockerfile, GitHub, container, Liberty, Helm, Dockerfile-lang, local directory, import, pre-import, pom.xml, server.xml, Maven, Docker, Helm chart
+duration: 1 minute
+permalink: mdteclipseimportedprojects
+type: document
+parent: importingaproject
+order: 0
+---
+
+# Importing projects
+
+Modifications are usually required to successfully add and deploy projects that have never been run in Codewind before. The add process creates required files if they do not exist. This guide covers the basics of configuring a project to run in Codewind.
+
+## What kind of projects can I add?
+
+Codewind is designed to develop cloud native microservices. Therefore, each project must be self-sufficient and not dependent on other projects to build. The requirements to add projects for each of the supported application types are outlined in the following sections:
+
+* [Eclipse MicroProfile projects](#eclipse-microprofile-projects)
```

Comment:
Not sure why we're calling them _Eclipse_ MicroProfile projects, in the IDEs they're called `MicroProfile / Java EE` projects

Reply:
Changed to `MicroProfile / Java EE projects`